backend/app/database/falkor/code-graph-backend/api/analyzers/python/analyzer.py

```
@@ -10,6 +10,7 @@
 
 import tree_sitter_python as tspython
 from tree_sitter import Language, Node
+from tree_sitter import QueryCursor, Query, Node
```

Ruff (0.14.11)

Redefinition of unused `Node` from line 12: `Node` redefined here

Remove definition: `Node`

(F811)